mig-agent/config.go

```diff
@@ -167,9 +183,30 @@ type globals struct {
 
 	// Add the private client key below.
 	agentKey []byte
+
+	// let you target specific agents
+	tags map[string]string
+}
+
+func globalTagsMap() map[string]string {
+	tags := make(map[string]string)
+	inspectTags := reflect.TypeOf(&TAGS).Elem()
+
+	for tag, val := range structs.Map(TAGS) {
+		field, ok := inspectTags.FieldByName(tag)
+
+		if ok {
+			_, tag = stringPair(string(field.Tag))
+			tag = strings.Replace(tag, `"`, "", -1)
+		}
+		tags[tag] = fmt.Sprintf("%v", val)
+	}
+	return tags
```

Comment:
I'm all in favor of converting the TAGS to a map! I will test that out and make sure it works out ok!